pkg/asset/installconfig/aws/permissions.go

```diff
@@ -57,6 +57,9 @@ const (
 	// PermissionKMSEncryptionKeys is an additional set of permissions required when the installer uses user provided kms encryption keys.
 	PermissionKMSEncryptionKeys PermissionGroup = "kms-encryption-keys"
 
+	// PermissionAMIEncryptionKeys is an additional set of permissions required when the installer uses an encrypted AMI.
```

Comment:
```suggestion
	// PermissionAMIEncryptionKeys is an additional set of permissions required when the installer uses an encrypted AMI with a KMS Customer Managed Key.
```

According to https://github.com/openshift/installer/pull/9921#discussion_r2321270753, this only applies for amis encrypted with CMK right?